irods_docker_files/setup_database_client.py

```diff
@@ -114,24 +136,13 @@ def install_oracle_dependencies():
     except KeyError:
         irods_python_ci_utilities.raise_not_implemented_for_distribution()
 
-def install_mysql_pcre(dependencies, mysql_service):
```

Comment:
we don't need this anymore?    i can't remember...   gives us the regex search...

Reply:
It is not referenced anywhere in the repository. If we need to use it, I can put it back, but perhaps in a library-esque location.

Reply:
oh... it was so we could do the resource hierarchy searching.   and now we do things by ID... so yeah, maybe we're good to go.